server/datastore/mysql/software.go

```diff
@@ -514,12 +505,17 @@ func selectSoftwareSQL(opts fleet.SoftwareListOptions) (string, []interface{}, e
 
 	ds = ds.GroupBy(
 		"s.id",
-		"scv.cpe_id",
+		"s.name",
+		"s.version",
+		"s.source",
+		"s.bundle_identifier",
+		"s.release",
+		"s.vendor",
+		"s.arch",
```

Comment:
Why are these being added to the group by?

Reply:
The previous statement was non-standards compliant and only works on MySQL - not that it matters much because we only target MySQL but, it was kind of bugging me :P